corehq/messaging/scheduling/static/scheduling/js/dashboard.js

```diff
@@ -1,6 +1,10 @@
-/* globals d3, nv, ko */
-hqDefine("scheduling/js/dashboard", function () {
-    var dashboardUrl = hqImport("hqwebapp/js/initial_page_data").reverse("messaging_dashboard");
+hqDefine("scheduling/js/dashboard",[
+    'jquery',
+    'knockout',
+    'hqwebapp/js/initial_page_data',
+    'nvd3/nv.d3.min',
+], function ($, ko, initialPageData) {
```

Comment:
The body of this module references both `d3` and `nv`, so both of them should appear in the parameter list here rather than expecting globals to be there.

Reply:
Done